src/site/xdoc/server/config.xml

```diff
@@ -29,10 +29,17 @@
   
     <p>All configuration files resides in the ./conf and ./conf/META-INF folder.</p>
     
-    <p>We ship with default configuration (embedded in jars) and -template files
+    <p>With default Spring packaging, we ship with default configuration (embedded in jars) and -template files
        you can use to override the default configuration. Simply copy in conf folder *-template.xml
        to *.xml (example: smtpserver-template.xml to smtpserver.xml), and change
        in *.xml the settings to map your needs.</p>
+
+    <p> For Guice based packaging, you are encouraged to take a look at default configuration of the
+      <a href="https://github.com/apache/james-project/tree/master/dockerfiles/run/guice">James docker images</a>.</p>
+
+    <p>Please note that configuration is dependent of the packaging you chose. Some configuration files are specific to
```

Comment:
Please note that the configuration is dependent of the packaging you have chosen.